allennlp/tests/fixtures/language_model/experiment_unidirectional_unsampled.jsonnet

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
It would be great to reduce the duplication of this config with that in allennlp/tests/fixtures/bidirectional_lm/experiment_unsampled.jsonnet and allennlp/tests/fixtures/language_model/experiment_unsampled.jsonnet.